app/views/triage/_patient_triage_card.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="nhsuk-card">
+  <div class="nhsuk-card__content">
+    <h2 class="nhsuk-card__heading nhsuk-heading-m">
+      Triage
+    </h2>
+
+    <%= form_for [session, triage], builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
```

Comment:
Do you need to specify the builder? It should be the default, I didn't have to on my branch...

Reply:
Nope, you're right I don't. Removed.